ambry-server/src/integration-test/java/com.github.ambry.server/RouterServerTestFramework.java

```diff
@@ -150,7 +150,7 @@ OperationChain startOperationChain(int blobSize, int chainId, Queue<OperationTyp
     TestUtils.RANDOM.nextBytes(data);
     short accountId = Utils.getRandomShort(TestUtils.RANDOM);
     short containerId = Utils.getRandomShort(TestUtils.RANDOM);
-    BlobProperties properties = new BlobProperties(blobSize, "serviceid1", accountId, containerId);
+    BlobProperties properties = new BlobProperties(blobSize, "serviceid1", accountId, containerId, false);
```

Comment:
Randomize the `encrypted` property? Or are you going to add tests here once the encryption support is available in router?

Reply:
yes.